DEVELOPMENT.md

```diff
@@ -55,6 +55,13 @@ development command line options.
   If handling of metadata has changed while developing, set this env var to
   `clear` to have cache `clear()`ed before use.
 
+- `DANDI_INSTANCEHOST` -- defaults to `localhost`. Point to host/IP which hosts
+  a local instance of dandiarchive.
+
+- `DANDI_REUSE_LOCAL_DOCKER_TESTS_API_KEY` -- use to make `local_docker_compose*`
```

Comment:
Suggested description: "Make the `local_docker_compose*` fixtures use the given API key for `DANDI_INSTANCEHOST` instead of spinning up a new environment with a new key."

Reply:
done, thanks! pushed